README.md

```diff
@@ -27,10 +27,10 @@ a different table, you'll need to add further permissions to the Cloudformation
 
 ### Testing your changes
 
-Ensure you are on the correct AWS region (`eu-west-1`). This can be achieved by using the `AWS_DEFAULT_REGION` environment variable:
+Ensure you are on the correct AWS region (`eu-west-1`). This can be achieved by using the `AWS_REGION` environment variable:
```

Comment:
@alexduf AWS_DEFAULT_REGION was not working for me, after googling I found that the variable name is AWS_REGION, which infact worked.